services/core-api/app/api/users/minespace/resources/minespace_user.py

```diff
@@ -42,6 +42,22 @@ def post(self):
             new_mum.save()
         return new_user
 
+    @api.doc(description='Update an existing Minespace Users mine list')
+    @api.marshal_with(MINESPACE_USER_MODEL)
+    @requires_role_mine_admin
+    def put(self, user_id):
```

Comment:
The PUT method should be contained within the MinespaceUserResource rather than the MinespaceUserListResource class given how we have the namespaces configured. The list resources typically handle GET ALL and POST, whereas the base resource handles operations on a single entity such as GET, PUT, DELETE.

Additionally you will need to add in the request parser into the MinespaceUserResource, much the same way that it is being done in the MinespaceUserListResource. 

Let me know if you need a walkthrough or have any questions.